### DISCUSSION TITLE
Important Thread (Please Read!)

### DISCUSSION THREAD

**Discussion by solomonshalom** [solomonshalom/BLC#49] [Announcements]
Hi!

I am Solomon, the lead developer of Bublr and the only maintainer 😅 (We do have contributors, for whom I really appreciate, be it the time and effort spent - gotta love them for it! :D) Though, this posts going to be a little different, hopefully not discouraging.

In the past few months, everyone on the team has been actively preparing for our academic test, which is a very important one too! Though, due to the ongoing development of Bublr, it's quite challenging to commit (pun-intended? maybe 😏) any time to our tests and so until December (or a little later, but do note), any and every PR done to Bublr will be in hold - be it a review or a merge.

I know that this isn't exactly great news, especially for those who are contributing to hacktoberfest and so, every commit until today (Oct 20th) will be reviewed // accepted!

P.S. If you have any Qs, shoot them below, and I will try to answer them! (24~)

---

Tagging those actively working on the codebase - @jennifershinshin, @MihirKohli, @rajanarahul93, @harshkas4na, and @ashishk1331.

**Comment by solomonshalom** [comment]
Hey there, @jennifershinshin, 

if you're reading this (the one above), don't get discouraged!

I know you've put in a ton of time and effort into Bublr, and I don't want to be the reason you hit the brakes. I seriously appreciate everything you've done and keep doing for Bublr and as a small token of my gratitude, I'm handing you the contributor badge :D

Even if I cannot merge your PRs, you can, and if you're up for it, others' too!

I get it, life can be crazy busy. If you are super held up, merge your PRs. But if you ever want to dive into more, the door's wide open! 🌊 

Super excited to see what you do!🚀

P.S. I think you should be getting the invitation for the role in your email, am not so sure but do let me know if I gotta redo it! ALSO, please know that this isn't binding - if you got the time and want to do it, it would truly be an honor!

**Comment by jennifershinshin** [comment]
Thank you! As someone new to open source, I got really lucky to come across your project!
I appreciate you 🙏

**Reply by solomonshalom** [reply]
The honour is ours, thank you so muchh! ✨
